data/load-seed-data.js

```diff
@@ -38,8 +38,8 @@ async function run() {
     await Promise.all(
       temprData.map(tempr => {
         return client.query(`
-                    INSERT INTO temprs (id, tempr_name)
-                    VALUES ($1, $2);
+                    INSERT INTO temprs (id, tempr_id, tempr_name)
```

Comment:
If I'm correct we would need the same thing here. (tempr_id, tempr_name) or (breed_id, tempr_name)